redo github

jgalber opened the pull request.

Commit:
redo github

tylervanbrocklin commented on scripts/script.js:
```diff
@@ -0,0 +1,53 @@
+function cost() {
+    alert("$123.45");
+}
+
```
You are on the right track here, but not quite what we're looking for. Remember, we want to show the cart total. So if you click two dogs, it should display $246.90". You can do this by creating a variable to keep track of cost.

tylervanbrocklin replied:
Also, you'll want to use function parameters. So something like: `func total(cost, dog, breed)` then when you call the function in your HTML code, you'd pass in the values.

jgalber replied:
Thank you so much Tyler, I have already pulled the project again with the committed changes.

tylervanbrocklin commented on scripts/script.js:
```diff
@@ -0,0 +1,53 @@
+
+function infoMurphy(){
+    alert("Murphy \nBreed: Boxer \n\nAdoption fees: $123.45");
+}
```
Functions are designed to be reused. In this case, you should only have one info function that works for all dogs. You'd do this by creating a function with info: `func total(cost, dog, breed)`

jgalber replied:
Great!! now, that makes sense, less and more effective code. As you suggested I change the name of the function and add some values, parameters and arguments, to wrap the function. Forgot to mention the sum of the totals when clicking the 'adopt' button. Thanks for your help.

Commit:
last update, js and html fixed

Commit:
Course Project - Media Queries

jgalber committed:
Update README.md

tylervanbrocklin commented on styles/style.css:
```diff
@@ -0,0 +1,1147 @@
+
+
+/* MEDIA QUERIES */
+@media screen and (min-width: 0px) and (max-width: 576px) {
```
Great work with the media queries. Note you do not need to specify a min-width here. If you just set a max-width, the compiler will allow for anything under your max-width.

jgalber replied:
Thanks Tyler. I was honestly not sure enough to define the 'minimum width' with the value of 0. I thought this idea will keep the layout strictly accurate to the specified style with the root value of 0. So some best practices recommend 'minimum widths' rather than 'maximum widths', but it makes sense here. Thanks again for your help, sometimes simpler is better.

tylervanbrocklin commented on styles/style.css:
```diff
@@ -0,0 +1,1147 @@
+
+@media screen and (max-width: 576px) {
+    .blog_content {
+        display: none;
```
nicely done getting the blog content to disappear on small screen sizes

Commit:
responsive images, css media queries

jgalber committed:
Update README.md

jgalber committed:
Update index.html

Some links needed to be fixed.

jgalber committed:
Update README.md

jgalber committed:
Update README.md

jgalber committed:
Update README.md

Commit:
styles gradient header updated

Commit:
Merge branch 'master' of https://github.com/jgalber/html200-adopt-a-dog into master

Commit:
Lesson 08 Assignment

Commit:
index has been updated

jgalber committed:
Update README.md

jgalber committed:
Update index.html

Fixed some errors

tylervanbrocklin commented on scripts/main.js:
```diff
@@ -0,0 +1,81 @@
+    
+];
+
+    for (let i = 0; i < fields.length; i++) {
```
Great work dynamically generating the blog posts. In the next course in the series, vue.js will make this much easier. Your code looks great

tylervanbrocklin commented on scripts/form.js:
```diff
@@ -0,0 +1,41 @@
+const form = document.getElementById('form');
+
+form.addEventListener('submit', function(event){  
```
Your form outputs in console and displays an alert as required. Nice job!

Commit:
milestone 3 assignment

Commit:
milestone 3 assignment

jgalber committed:
Update index.html

jgalber committed:
Update index.html

jgalber committed:
Update README.md

Commit:
dog tiles hover effect fixed

jgalber committed:
Update index.html

jgalber committed:
Update index.html

jgalber committed:
Update script-query.js

jgalber committed:
Update dogs.html

jgalber committed:
Update index.html

jgalber committed:
Update blog.html

jgalber committed:
Update dogs.html

jgalber committed:
Update script-query.js

jgalber committed:
Update script-query.js

jgalber committed:
Update script-query.js

jgalber committed:
Update script-query.js

jgalber committed:
Update index.html

jgalber committed:
Update script-query.js

jgalber committed:
Update script-query.js

jgalber committed:
Update index.html

jgalber committed:
Update script-query.js

jgalber committed:
Update script-query.js

jgalber committed:
Update index.html

jgalber committed:
Update dogs.html

jgalber committed:
Update script-query.js

jgalber committed:
Update script-query.js

jgalber committed:
Update dogs.html

jgalber committed:
Update dogs.html

jgalber committed:
Update dogs.html

Commit:
cart updated

Commit:
cart updated

Commit:
cart updated

Commit:
updated

Commit:
style fixed cart for medium and small devices

jgalber committed:
Update script-query.js

Alert message when submitting the form fixed

jgalber committed:
Update README.md

jgalber committed:
Rename README.md to weekly-summary.md

jgalber committed:
Adopt A Dog, synopsis - Lesson 10 assignment

jgalber committed:
Add files via upload

jgalber committed:
Updating Readme File

jgalber committed:
upload second file of course project assignment

This document includes two user-persona and the result of Adopt A Dog project test.
